src/Html.php

```diff
@@ -2129,12 +2134,41 @@ public static function popHeader(
         $HEADER_LOADED = true;
 
         self::includeHeader($title, $sector, $item, $option); // Body
-        echo "<body class='" . ($iframed ? "iframed" : "") . "'>";
+        echo "<body class='" . ($in_modal ? "in_modal" : "") . "'>";
         self::displayMessageAfterRedirect();
         echo "<div id='page'>"; // Force legacy styles for now
     }
 
 
+    /**
+     * Print a nice HTML head for iframed windows
+     *
+     * @since 10.0.7
+     *
+     * @param string  $sector    sector in which the page displayed is (default 'none')
+     * @param string  $item      item corresponding to the page displayed (default 'none')
+     * @param string  $option    option corresponding to the page displayed (default '')
+     * @return void
+     */
+    public static function iframedHeader(
+        string $sector = "none",
+        string $item = "none",
+        string $option = ""
```

Comment:
As it lowers the security, it should not be done by default.
```suggestion
        string $option = "",
        bool $allow_embedding = false
```

Reply:
what the point of this one ?

The new header is specifically dedicated to allow an iframe.
Before the last commit, i used popHeader, and i agreed it exposed `in_modal` current usage to not have the SAMEORIGIN header.
This new method permits to opt in a iframe usage, so i don't understand the need of a parameter, in case of false, just use popHeader.

Reply:
I think that with a name like `iframedHeader`, people will probably use this without even know that it allow embedding outside GLPI.